modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/OpenIdConnectUserInfoProcessorImpl.java

```diff
@@ -137,32 +176,6 @@ private void _checkAddUser(long companyId, String emailAddress)
 		}
 	}
 
-	private long[] _getRoleIds(long companyId) {
```

Comment:
You probably don't want to remove this method since Brian extract your code to it. You may want to do isser test and then call this method.

Reply:
Hey @alvarosaugarlr 
I am off today, will check the PR tomorrow. Please address this issue and once you have it ready, run ci:test:sf and ci:test:relevant

Reply:
You are right. I'm going to change it and commit it.